src/Document.tsx

```diff
@@ -151,27 +177,32 @@ const Document = (props: DocProps) => {
                   </>
                 )}
               </Box>
+              <Box>
```

Comment:
Ideally we'd keep the spacing consistent between each of these sections.
<img width="489" alt="image" src="https://user-images.githubusercontent.com/66292936/107854178-1d00f200-6e12-11eb-9de5-26ed44f8e30b.png">
I'd suggest using the same <Divider /> between each section.

Reply:
Do we want a divider (like between `Anchoring` and `Metadata`)  between each section

Reply:
Yes, exactly!